client/src/components/EndpointCardHeader/index.tsx

```diff
@@ -1,43 +1,43 @@
 import classnames from 'classnames';
 import React, { memo } from 'react';
 
-import { EndpointLogo } from '~/components/EndpointLogo';
 import { ServiceCard } from '~/domain/service-card';
 
+import { EndpointLogo } from './EndpointLogo';
+
 import css from './styles.scss';
 
 export interface LayerProps {
   card: ServiceCard;
+  currentNamespace: string | null;
 }
 
 export const EndpointCardHeader = memo(function EndpointCardHeader(
   props: LayerProps,
 ) {
   const card = props.card;
 
+  const showSubtitle =
+    Boolean(card.namespace) &&
```

Comment:
I strongly recommend using `!!` (double negation) for boolean coercion as it have better performance and readability